.gitignore

```diff
@@ -0,0 +1 @@
+build_artefacts
```

Comment:
Appveyor calls them [artifacts](https://www.appveyor.com/docs/packaging-artifacts/), so does [travis](https://docs.travis-ci.com/user/uploading-artifacts/) and [circleci](https://circleci.com/docs/build-artifacts/).

Reply:
Addressing in the feedstocks and staged-recipes with PRs ( https://github.com/conda-forge/conda-smithy/pull/638 ) and ( https://github.com/conda-forge/staged-recipes/pull/4753 ).